### DISCUSSION TITLE
Should I merge or integrate my samples?

### DISCUSSION THREAD

**Discussion by drewvoigt10** [stuart-lab/signac#667] [Q&A] [answered]
Thanks for the awesome package. After using Seurat for many scRNA-seq experiments, our group has recently performed our first sc-ATAC seq experiment, and we will be analyzing the data in Signac. 

We are wondering about best practices for data integration and/or merging. We have two (human) control samples (control-1 and control-2), and two diseased samples (disease-1 and disease-2). These samples were barcoded on the same 10X genomics run and pooled for sequencing. Of note, we recovered ~5K cells in the control samples but only 1-2K in the diseased samples. 

If these were RNA samples, I would use Seurat's canonical correlation analysis to integrate all of the samples together. Since this is our first time dealing with the ATAC data, I am wondering if we should INTEGRATE the four ATAC samples (following the [integration vignette](https://satijalab.org/signac/articles/integrate_atac.html)) or MERGE the four samples (following the [merging object vignette](https://satijalab.org/signac/articles/merging.html)). Any insight would be very helpful - thank you in advance!

**Comment by timoast** [comment, marked as answer]
Whether you should merge or integrate data depends on whether there is technical differences between datasets (eg, a batch effect) that you would like to remove.

I would suggest first processing the data without integration (quantify the same peak set in each dataset, merge, run LSI, run UMAP, etc) and assessing whether you think there is batch-to-batch variation that needs to be removed. If so, you can follow the examples in the integration vignette on the Signac website.
